addons/crm/wizard/crm_lead_pls_description.xml

```diff
@@ -0,0 +1,34 @@
+<odoo>
+    <record model="ir.ui.view" id="crm_lead_pls_description_form_view">
+        <field name="name">crm.lead.pls.description.wizard.form</field>
+        <field name="model">crm.lead.pls.description</field>
+        <field name="arch" type="xml">
+            <form string="PLS Description">
+                <div><label for="sale_team_id"/><field name="sale_team_id"/></div>
+                <div><label for="closed_leads_count"/><field name="closed_leads_count"/></div>
+                <div><label for="win_probability"/><field name="win_probability"/></div>
```

Comment:
I would put `:` between label and value.
too look like
**Sales Team :** Europe
**Number of leads of this team :** 4 
**Team winning rate (%) :** 25

I think this would be more clear